Cargo.lock

```diff
@@ -1292,7 +1292,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4271d37baee1b8c7e4b708028c57d816cf9d2434acb33a549475f78c181f6253"
 dependencies = [
  "fallible-iterator",
- "indexmap 2.2.3",
+ "indexmap 2.2.5",
```

Comment:
Please revert this commit. Dependency updates require reviewing the changes in those dependencies by a trusted reviewer, which cargo-vet enforces. Because no such review has been done, CI would fail once it runs.